.gitignore

```diff
@@ -3,4 +3,5 @@
 *.swp
 *.pyc
 *.log
-.env
\ No newline at end of file
+.env
+/.idea/
```

Comment:
Why you added this? Irrelevant to the issue

Reply:
@vaibhavsingh97 I used the command `git rm --cached .gitignore` so that this file doesnot get pushed but still it did. 
what should I do to remove this ?